qutebrowser/browser/webengine/webengineelem.py

```diff
@@ -224,6 +224,16 @@ def _requires_user_interaction(self) -> bool:
         ):
             return True
 
+        # Qt 6.3+ needs a user interaction to allow navigations from file:// to
+        # outside file:// (like e.g. on file://bookmarks.html).
+        versions = version.qtwebengine_versions()
```

Comment:
There's a bit much duplication here - could at minimum get rid of this, but I'd probably do the whole thing in a `for scheme in ["qute", "file"]:` for loop instead?